### PR TITLE
Publish AWS deployment samples

### PR DESCRIPTION
**Pull request by dcbroad3**

### CODE REVIEW

**Review by derekendres — APPROVED** [review]

**Comment by derekendres** [comment]
I will wait for Chris to also review before merge

**Comment by cholzmer** [comment]
> 
> 
> I will wait for Chris to also review before merge

I'll start looking at it now. Thanks.

**Review by cholzmer — APPROVED** [review]

**Pull request merged**
